### DISCUSSION TITLE
Missing qt.core.plugin.factoryloader: /plugins/styles directory

### DISCUSSION THREAD

**Discussion by pgupta-quantumSi** [miurahr/aqtinstall#804] [Q&A]
I am using aqtinstall to install Qt 6.7 on the ubuntu 20.0 distribution. 
`# Install Qt using aqtinstall
RUN pip3 install aqtinstall

ARG QT=6.7.0
ARG QT_HOST=linux
ARG QT_TARGET=desktop
ARG QT_ARCH=linux_gcc_64
ARG QT_OUTPUT_DIR=/usr/local/Qt6
#/usr/share/

RUN mkdir -p ${QT_OUTPUT_DIR}
RUN aqt install-qt --outputdir ${QT_OUTPUT_DIR} ${QT_HOST} ${QT_TARGET} ${QT} ${QT_ARCH}`

Installation succeeds, however at the end when I am running an application I get the following error

`qt.core.plugin.factoryloader: Got keys from plugin meta data QList("ico", "cur")
qt.core.plugin.factoryloader: checking directory path "/home/instrumentControlUI/imageformats" ...
qt.core.library: "/usr/local/Qt6/6.7.0/gcc_64/plugins/imageformats/libqjpeg.so" loaded library
qt.core.library: "/usr/local/Qt6/6.7.0/gcc_64/plugins/imageformats/libqsvg.so" loaded library
qt.core.library: "/usr/local/Qt6/6.7.0/gcc_64/plugins/imageformats/libqgif.so" loaded library
qt.core.library: "/usr/local/Qt6/6.7.0/gcc_64/plugins/imageformats/libqico.so" loaded library
Cannot find EGLConfig, returning null config
qt.core.plugin.factoryloader: checking directory path "/usr/local/Qt6/6.7.0/gcc_64/plugins/styles" ...
Required extensions missing!
Aborted
`
I checked the `/usr/local/Qt6/6.7.0/gcc_64/plugins/styles` directly but there is no styles plugin installed under plugins directory.
Additionally, I have opengl dependencies already installed as I can run any Qt5 application without any problem. 

Could you suggest if I need to install any other dependcies?

**Comment by pzhlkj6612** [comment]
Hi. Here I'm not going to answer your question, but share a tech tip: [Creating and highlighting code blocks - GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks). You were [quoting code](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) with backticks. They are not the same.
